201-cdn-with-ruleseengine-cacheoverride/README.md

```diff
@@ -0,0 +1,20 @@
+# Simple deployment of cache override via rules engine for a CDN end point
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/PublicLastTestDate.svg" />&nbsp;
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/PublicDeployment.svg" />&nbsp;
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/FairfaxLastTestDate.svg" />&nbsp;
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/FairfaxDeployment.svg" />&nbsp;
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/BestPracticeResult.svg" />&nbsp;
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/CredScanResult.svg" />&nbsp;
```

Comment:
Not sure how to create the images. All examples have these images but don't see any documentation on how to upload the images.